ci/playbooks/kuttl/deploy-deps.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Run ci_framework bootstrap playbook
+  ansible.builtin.import_playbook: "../../../playbooks/01-bootstrap.yml"
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  name: install dev tools
+  tasks:
+    - name: Download install_yamls deps
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_download_tools'
+
+- name: Run ci_framework infra playbook
+  ansible.builtin.import_playbook: "../../../playbooks/02-infra.yml"
+
+- name: Build dataset hook
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Assert that operator_name is set
+      ansible.builtin.assert:
+        that:
+          - operator_name is defined
+
+    - name: Load parameters
+      ansible.builtin.include_vars:
+        dir: "{{ item }}"
+      loop:
+        - "{{ cifmw_basedir }}/artifacts/parameters"
+        - "/etc/ci/env"
+      loop_control:
+        label: "{{ item }}"
+
+    - name: Check we have some compute in inventory
+      ansible.builtin.set_fact:
+        has_compute: >-
+          {% set ns = namespace(found=false) -%}
```

Comment:
No need to use a namespace, a regular var is enough. Moreover, this can be done without jinja2 scripts with plain ansible-jinja2 filters:
```yaml
hostvars.keys() | select('search', '^compute.*') | length > 0

```

Reply:
actually we don't set a compute in the nodeset for this job. I carried this from the nova kuttl job, but it must be there for a different reason, since the kuttl nodeset does not have compute nodes. I'm removing this task